src/main/java/BootApp.java

```diff
@@ -1,23 +1,19 @@
-import configuration.ConfigurationYAML;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.ComponentScan;
+import configuration.TriggerEndpointConfiguration;
+import configuration.UserConfiguration;
+import com.google.common.collect.ImmutableList;
```

Comment:
My intellij doesn't have this package. Gradle clean build doesn't help. Shouldn't it be in a gradle file?

Reply:
this is from guava, please refresh your gradle project and let gradle download the dependency

Reply:
it's being added in this file
https://github.com/KacperKenjiLesniak/Server-Push/blob/eb61ca7076885a7ef9da9815f39aa3bdc6204427/build.gradle

Reply:
OK, I had to change from local gradle to gradle wrapper. It works fine now :)